.circleci/config.yml

```diff
@@ -1,22 +1,161 @@
+    steps:
+      - deploy:
+          command: |
+            cd /tmp/artifacts
+            svn st | grep ^! | awk '{print " --force "$2}' | xargs svn rm
+            svn add --force .
+  svn_create_tag:
+    description: "Create a SVN tag"
+    steps:
+      - deploy:
+          command: |
+            cd /tmp/artifacts
+            VERSION=grep 'Version:' /tmp/src/plugin.php | awk -F: '{print $2}' | sed 's/^\s//'
+            svn cp trunk tags/$VERSION
+  svn_commit:
+    description: "Commit changes to SVN"
+    steps:
+      - deploy:
+          command: |
+            cd /tmp/artifacts
+            VERSION=grep 'Version:' /tmp/src/plugin.php | awk -F: '{print $2}' | sed 's/^\s//'
+            svn ci -m "Tagging $VERSION from Github" --no-auth-cache --non-interactive --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+
+executors:
+  base:
+    docker:
+      - image: circleci/buildpack-deps:latest
+    working_directory: /tmp
+  php_node:
     docker:
       - image: circleci/php:7.3.3-stretch-node-browsers
+    working_directory: /tmp/src
+
+jobs:
+  checkout:
+    executor: base
     steps:
-      - checkout
-      - prepare-environment
+      - checkout:
+          path: src
+      - persist_to_workspace:
+          root: /tmp
+          paths:
+            - src
+
+  checks:
+    executor: php_node
+    steps:
+      - attach_workspace:
+          at: /tmp
+      - install_dependencies
       - run: composer phpcs
+      - persist_to_workspace:
+          root: /tmp
+          paths:
+            - src
 
-commands:
- prepare-environment:
-   description: "Install dependencies."
-   steps:
-     - run: composer install
+  deploy_svn_branch:
+    executor: base
+    steps:
+      - attach_workspace:
+          at: /tmp
+      - svn_setup
+      - svn_add_changes
+      - svn_commit
+
+  deploy_svn_tag:
+    executor: base
+    steps:
+      - attach_workspace:
+          at: /tmp
+      - svn_setup
+      - svn_add_changes
+      - svn_create_tag
+      - svn_commit
 
 workflows:
   version: 2
-  check-wp-cs:
+  checks:
+    jobs:
+      - checkout
+      - checks:
+          requires:
+            - checkout
+
+  branch_deploy:
+    jobs:
+      - checkout:
+          filters:
+            branches:
+              only: master
+      - checks:
+          requires:
+            - checkout
+          filters:
+            branches:
+              only: master
+      - deploy_svn_branch:
+          context: genesis-svn
+          requires:
+            - checkout
+            - checks
+          filters:
+            branches:
+              only: master
+
+  tag_deploy:
     jobs:
-      - test
+      - checkout:
+          filters:
+            tags:
+              only: /^\d+\.\d+\.\d+$/
+            branches:
+              only: master
+      - checks:
+          filters:
+            tags:
+              only: /^\d+\.\d+\.\d+$/
+            branches:
+              only: master
+          requires:
```

Comment:
My eyes went cross wondering why this 'checks' job looked different.  Only cause 'requires:' is listed after 'filters:'.  The others have 'filters:' after 'requires:'.  Not a blocker.